### PR TITLE
Makes alcoholic beverages make more sense

### PR DESCRIPTION
**Pull request by pantherpusher**
Makes certain beverages make more sense based on how much booze is actually in them

## About The Pull Request

My sanity is draining and I wanted to make the alcohol 

## Why It's Good For The Game

Makes more sense

## Changelog
:cl:
tweak: tweaked the strength values of some drinks in hopes of making them make more sense
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagents/food-Drinks.dm [hunk 1]
```diff
@@ -2021,7 +2021,7 @@
 	taste_description = "a numbing sensation"
 	reagent_state = LIQUID
 	color = "#2E2E61"
-	strength = 10
+	strength = 3
```
Comment (EvilJackCarver) [inline code/modules/reagents/reagents/food-Drinks.dm:2024]: Is it your intent here to make Neurotoxin stronger than the game's strongest alcohol, which is intentionally kept behind a glass case?

Comment (pantherpusher) [reply code/modules/reagents/reagents/food-Drinks.dm:2024]: I believe you misunderstand, the pan-galactic gargle blaster has a strength of 1, hooch has a strength of 1, atomic vodka has a strength of 5 with a multiplier of 10 which I assume makes it 10 times stronger if you don't have the perk that makes you immune to it, I could be wrong but there are already things stronger than atomic vodka based off their strength number alone

Comment (EvilJackCarver) [reply code/modules/reagents/reagents/food-Drinks.dm:2024]: Ah, I missed that.

**Review by EvilJackCarver — APPROVED** [review]
Fix up your changelog and you'll be good to go.

**Comment by MciReloaded** [comment]
I'm curious- does as higher number mean stronger or a lower number mean stronger? Because either way, I am confused as to how you got some of these numbers. Is it done by ABV, how quick they get into the system, or something else?

**Comment by EvilJackCarver** [comment]
> 
> 
> I'm curious- does as higher number mean stronger or a lower number mean stronger? Because either way, I am confused as to how you got some of these numbers. Is it done by ABV, how quick they get into the system, or something else?

It acts like a divisor. Lower number is stronger.

**Pull request merged**